data/korath/korath jobs.txt

```diff
@@ -243,3 +243,588 @@ mission "Kor Efreti to Wanderers: Farming"
 	on complete
 		payment 10000 200
 		dialog `When you return, the Korath move their hands in a circular motion with their palms toward you as a sign of gratitude. They hand you <payment> and disembark your ship.`
+
+mission "Korath Humanika: Fish Trade (small)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 5 10 0.8
+	illegal 50000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (small): active"
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 25000 500
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" ++
+
+mission "Korath Humanika: Fish Trade (medium)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 15 50 0.8
+	illegal 100000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 20
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (medium): active"
+		"cargo space" > 20
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 35000 400
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" += 2
+
+mission "Korath Humanika: Fish Trade (bulk)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 40 300 0.8
+	illegal 150000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 5
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (bulk): active"
+		"cargo space" > 50
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 45000 350
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" += 3
+
+mission "Korath: Retrieve livestock [0]"
+	name "Livestock from <stopovers>"
```

Comment:
These titles clip pretty heavily with most planet/systems names right now, so I put up a PR for `<planet stopovers>` at https://github.com/endless-sky/endless-sky/pull/8467 that should make them fit much more.

![image](https://user-images.githubusercontent.com/4471575/221462209-8e9a7502-388a-47fb-8f57-e1e88f1159dc.png)

Reply:
I'd rather not have the code diverge in this branch, so I'll wait for the `<planet stopovers>` to reach master.

Reply:
PR's merged to master now, so next time you pull upstream changes, should be good to go for using that term.

Reply:
Excellent. I will work on that in my next batch of changes.